### PR TITLE
#2183 add progress unit test

### PR DESCRIPTION
**Pull request by DavidMi-1**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2183)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
@wu87988622 could you help review this PR? I don't know much about the progress bar component :)

**Comment by wu87988622** [comment]
> @wu87988622 could you help review this PR? I don't know much about the progress bar component :)

ok, i will review this

File: ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    const step2 = getAllByTestId(props.testId)[1].querySelector('svg');
+
+    step1.getAttribute('class').includes('MuiStepIcon-active');
+    expect(step2).toHaveAttribute('class', 'MuiSvgIcon-root MuiStepIcon-root');
+
+    getByText('1');
+    getByText('2');
+  });
+
+  it('renders progress active step 2/3', async () => {
```
Comment (wu87988622) [inline ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js]: renders progress active step 1/3

File: ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    step1.getAttribute('class').includes('MuiStepIcon-completed');
+    step2.getAttribute('class').includes('MuiStepIcon-active');
+    expect(step3).toHaveAttribute('class', 'MuiSvgIcon-root MuiStepIcon-root');
+
+    expect(queryByText('1')).toBeNull();
+    getByText('2');
+    getByText('3');
+  });
+
+  it('renders progress active step 3/3', async () => {
```
Comment (wu87988622) [inline ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js]: renders progress active step 2/3

File: ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { cleanup, render, waitForElement } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
```
Comment (wu87988622) [inline ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js]: in this case you does't use this dependency
Maybe uninstall dependency ?

File: ohara-manager/package.json [hunk 4]
```diff
@@ -60,6 +60,7 @@
     "yargs": "12.0.2"
   },
   "devDependencies": {
+    "@testing-library/react-hooks": "2.0.1",
```
Comment (wu87988622) [inline ohara-manager/package.json]: Can't you remove this dependency now?
This dependency only use in client

File: ohara-manager/client/package.json [hunk 5]
```diff
@@ -27,6 +27,7 @@
     "@testing-library/cypress": "4.0.4",
     "@testing-library/jest-dom": "4.0.0",
     "@testing-library/react": "8.0.7",
+    "@testing-library/react-hooks": "1.1.0",
```
Comment (wu87988622) [inline ohara-manager/client/package.json]: Maybe wait for the last dependency?

Comment (DavidMi-1) [reply ohara-manager/client/package.json]: ok

File: ohara-manager/package.json (hunk 4 above)
Comment (DavidMi-1) [reply ohara-manager/package.json]: ok

File: ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js (hunk 3 above)
Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Feedback/__tests__/Progress.test.js]: ok

**Pull request merged**